### PR TITLE
feat: new filterPrecursorCharge() method - see issue #192

### PR DESCRIPTION
**Pull request by lgatto**

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -8,25 +8,25 @@ Description: The Spectra package defines an efficient infrastructure
    backends tuned for fast data access and processing and backends for very
    large data sets ensuring a small memory footprint.
 Authors@R: c(person(given = "RforMassSpectrometry Package Maintainer",
-		    email = "maintainer@rformassspectrometry.org",
```
Comment (jorainer) [inline DESCRIPTION:11]: you need also to bump the version above to 1.1.19

File: NEWS.md [hunk 2]
```diff
@@ -2,6 +2,10 @@
 
 ## Changes in 1.1.18
```
Comment (jorainer) [inline NEWS.md:3]:
```suggestion
## Changes in 1.1.19
```

**Review by jorainer — CHANGES_REQUESTED** [review]
Looks good @lgatto ! Thanks for the PR - only change request is to bump the version of the package. After that you're free to merge.

File: DESCRIPTION (hunk 1 above)
Comment (lgatto) [reply DESCRIPTION:11]: Ah yes, forgot that! Done now, thanks

File: NEWS.md (hunk 2 above)
Comment (lgatto) [reply NEWS.md]: fixed too

**Pull request merged**